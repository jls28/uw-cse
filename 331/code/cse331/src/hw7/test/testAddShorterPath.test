LoadGraph letters letters.tsv
AddEdge letters a y 0.0
FindPath letters a b