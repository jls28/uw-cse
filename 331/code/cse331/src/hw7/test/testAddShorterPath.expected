loaded graph letters
added edge 0.000 from a to y in letters
path from a to b:
a to y with weight 0.000
y to b with weight 1.000
total cost: 1.000